src/lib/components/composites/user-avatar/UserAvatar.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+    import * as Avatar from "$lib/components/primitives/avatar/index.js";
+    import { ReviewDecision, type User } from "$lib/types";
+    import Check from "lucide-svelte/icons/check";
+    import { X } from "lucide-svelte";
```

Comment:
I am not sure if this makes a difference:
```suggestion
    import X from "lucide-svelte/icons/x";
```

Reply:
I don't think so, but it looks better, so I will change it.